pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/api/LetterResource.kt

```diff
@@ -9,8 +9,8 @@ class ParseLetterDataException(msg: String, cause: Exception): Exception(msg, ca
 class LetterResource(val templateResource: TemplateResource = TemplateResource()) {
     private val objectMapper = jacksonObjectMapper()
 
-    fun create(letterRequest: VedtaksbrevRequest): Letter<*> {
-        val template: LetterTemplate<*, *> = templateResource.getVedtaksbrev(letterRequest.kode)
+    fun create(letterRequest: AutobrevRequest): Letter<*> {
+        val template: LetterTemplate<*, *> = templateResource.getAutoBrev(letterRequest.kode)
```

Comment:
Har du testet at det fungerer å bestille et brev med Unit som LetterArg? Jeg tar det ikke for gitt at `parseArgument` fungerer da.

Reply:
Ville det ikke feilet i testen "all templates can render and compile"?

Reply:
Nei, for da lages det Unit med `createLetterData` (eller hva den funksjonen heter)

Reply:
Den testen vil jo kjøre fra utenfor brevbakeren mot containeren og mot grensesnittet som kjører parseArgument på unit.

Reply:
Hva blir Unit serialisert til i json da?

Reply:
Kan legge til en unit test etterpå uansett så den ikke er avhengig av den andre.